x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/shared_fields/shrink_field.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiTextColor } from '@elastic/eui';
+import React, { FunctionComponent } from 'react';
+
+import { UseField, NumericField } from '../../../../../../shared_imports';
+
+import { useEditPolicyContext } from '../../../edit_policy_context';
+import { i18nTexts } from '../../../i18n_texts';
+
+import { LearnMoreLink, DescribedFormField } from '../../';
+
+interface Props {
+  phase: 'hot' | 'warm';
+}
+
+export const ShrinkField: FunctionComponent<Props> = ({ phase }) => {
+  const path = `phases.${phase}.actions.shrink.number_of_shards`;
+  const { policy } = useEditPolicyContext();
+  return (
+    <DescribedFormField
+      title={
+        <h3>
+          <FormattedMessage
+            id="xpack.indexLifecycleMgmt.editPolicy.shrinkText"
+            defaultMessage="Shrink"
+          />
+        </h3>
+      }
+      description={
+        <EuiTextColor color="subdued">
+          <FormattedMessage
+            id="xpack.indexLifecycleMgmt.editPolicy.shrinkIndexExplanationText"
+            defaultMessage="Shrink the index into a new index with fewer primary shards."
+          />{' '}
+          <LearnMoreLink docPath="indices-shrink-index.html#indices-shrink-index" />
```

Comment:
I know the docs link was there before, but this page seems to work better for this context, wdyt? 
https://www.elastic.co/guide/en/elasticsearch/reference/master/ilm-shrink.html